test/LoginStrategy.test.ts

```diff
@@ -0,0 +1,90 @@
+import assert from 'assert';
+import { config } from 'dotenv';
+import { Keys } from '@ew-did-registry/keys'
+import { ClaimsUser } from '@ew-did-registry/claims';
+import { ClaimData } from './claim-creation/ClaimData';
+import { ITokenPayload } from '../lib/LoginStrategy.types';
+import { ClaimsUserFactory } from './claim-creation/ClaimsUserFactory';
+import { LoginStrategy, LoginStrategyOptions } from '../lib/LoginStrategy';
+import { providers } from 'ethers';
+
+config();
+
+let firstClaimToken: string;
+let claimData: ClaimData;
+let payload: ITokenPayload;
+let claimsUser: ClaimsUser;
+let secondClaimToken: string;
+let otherclaimsUser: ClaimsUser;
+let loginProcess: LoginStrategy;
+let secondPayload: ITokenPayload;
+
+const initParams: LoginStrategyOptions = {
+    jwtSecret: process.env.PRIVATE_PEM,
+    jwtSignOptions: {
+      algorithm: 'RS256',
+    },
+    name: 'login',
+    rpcUrl: process.env.RPC_URL || 'https://volta-rpc.energyweb.org/',
+    cacheServerUrl: process.env.CACHE_SERVER_URL || 'https://identitycache-dev.energyweb.org/',
+    acceptedRoles: process.env.ACCEPTED_ROLES ? process.env.ACCEPTED_ROLES.split(',') : [],
+    privateKey: process.env.PRIVATE_KEY,
+  }
+
+const provider = new providers.JsonRpcProvider(initParams.rpcUrl)
+const keys = new Keys({
+    privateKey: process.env.PRIVATE_KEY,
+  });
+  const secondKeys = new Keys({
+    privateKey: process.env.PRIVATE_KEY_2,
+  });
+  const issuerDID = `did:ethr:${keys.getAddress()}`
+  const secondDID = `did:ethr:${secondKeys.getAddress()}`
+
+describe("Testing LoginStrategy", () => {
```

Comment:
Just a heads up, I think that the tests are good but as they are against deployed infrastructure (e.g. the volta RPC), I'm not sure they can be run during CI builds as deployed infrastructure may not always be available.

Reply:
@knzeng-e we can disable for ci by adding an `x` in front of the `describe`
```suggestion
xdescribe("Testing LoginStrategy", () => {
```